src/module.c

```diff
@@ -13032,6 +13052,51 @@ int VM_RdbSave(ValkeyModuleCtx *ctx, ValkeyModuleRdbStream *stream, int flags) {
     return VALKEYMODULE_OK;
 }
 
+/* Registers a new scripting engine in the server.
+ *
+ * - `engine_name`: the name of the scripting engine. This name will match
+ *   against the engine name specified in the script header using a shebang.
+ *
+ * - `engine_ctx`: engine specific context pointer.
+ *
+ * - `engine_methods`: the struct with the scripting engine callback functions
+ * pointers.
```

Comment:
Please mention that it returns OK/ERR on success/error and that something is logged when it fails.

Nit: Alignment typo.

```suggestion
 *   pointers.
```